modules/admin_manual/pages/configuration/server/occ_commands/app_commands/_wnd_commands.adoc

```diff
@@ -49,16 +49,32 @@ Listen to smb changes and store notifications for later processing in the databa
 | `-p, +
 --path=PATH` 
 | The path inside the share to watch for changes [default: ""]
+
 | `--password-file=PASSWORD-FILE`
 | The file containing the password for the account to be used to listen
+
+| `--password-from-service-account`
+| Use the password from the matching service account. This works only for collaborative WND mounts
+
 | `--password-trim` 
 | Trim blank characters from the password
+
 | `--unbuffering-option=UNBUFFERING-OPTION`
 | Force the usage of that unbuffering option for the underlying smbclient command. Possible options are either "auto", "pty" or "stdbuf" [default: "auto"]
+
 | `--output[=OUTPUT]`
 | The output format to use (plain, json or json_pretty). [default: "plain"] 
 |===
 
+[NOTE]
+====
+. To enter the password of the service account via the console, the `password` argument, the `password-file` option and the `password-from-service-account` option must not be set.
+
+. To use the password for the service account from the mount point settings, use the `password-from-service-account` option.
+
+. If you use the `password` argument together with the `password-file` option, the `password-file` option takes precedence over the `password` argument.
```

Comment:
Same as above: In this enumeration, one item is an "argument", the other two are "options"
```suggestion
. If you use the `password` option together with the `password-file` option, the `password-file` option takes precedence over the `password` argument.
```